app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/model/Movie.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.marcinwisniewski.intivetrainingapp.model
+
+data class Movie(var id: Int, var title: String, var year: String, var poster: String)
```

Comment:
why var? I think the class should be immutable, use val instead.

Reply:
Done